Topic1_Arrays/Day2207/TranVanThi/main.py

```diff
@@ -0,0 +1,18 @@
+# two sum
+
+def hashMap(nums, target):
```

Comment:
Please add the way you think to get the solution.
Add the time and space complexity